docs/low-code-sql/getting-started-with-low-code-sql.md

```diff
@@ -0,0 +1,382 @@
+
+![Create a new seed](img/6-1-1-create-a-new-seed.png)
+
+To create a seed click on the **(1) + Add Seed** button. A new pop-up window will appear where you can define metadata of the seed. There you can define the **(2) Name** of seed (which is going to be the same as the name of the table created) and the **(3) Path** for for it. When ready press **(4) OK,** to add.
+
+![Define Seed Value](img/6-1-2-define-seed-value.png)
+
+The seed initializes by default empty. To define the value simple copy and paste the content of desired CSV (each column separate by a comma, each row separated by a new line, with a header as the first line) into the **(1) Code editor.** To verify whether the seed is parsed correctly, you can see it after switching again to the **(2) Visual editor.**
+
+For the purpose of this tutorial, create a nation seed, with the following content:
+
+```n_nationkey,n_name,n_regionkey,n_comment
+0,ALGERIA,0,Vast Sahara; oil-rich.
+1,ARGENTINA,1,Tango; beef; Patagonia.
+2,BRAZIL,1,Amazon; Carnival; soccer.
+3,CANADA,1,Maple; multicultural; vast wilderness.
+4,EGYPT,4,Ancient pyramids; Nile River.
+5,ETHIOPIA,0,Coffee origin; diverse culture.
+```
+
+### 6.2 Define datasets
+
+Importing datasets is really easy. We can just drag-and-drop our existing tables directly into a model. We’re going to demonstrate that in the next step.
+
+## 7. Develop your first model
+
+A model is an entity that contains a set of data transformations and defines either a view or a table that will be created on the warehouse of choice. Each model is stored as a select statement in a SQL file within a project. Prophecy models are based on dbt Core [models](https://docs.getdbt.com/docs/build/models).
+
+![Create the first model](img/7-1-create-the-first-model.png)
+
+Here we create customers_nations model that’s going to enrich our customers and produce a report of which customers show up in which geographic areas most commonly.
+
+To create a new model simply click on the **(1) + Add Model** in the sidebar, when hovering over Models section, or **(2) Create Model** button. A model creation pop-up will show up, with very similar options, as when we defined the seed, available. Enter the **(3) Model Name** and **(4) Model Path** and that’s it. Finally save the model by pressing **(5) OK.**
+
+### 7.1 Drag and drop model’s graph
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://Prophecy-1.wistia.com/medias/wowmg8l0qq" title="Drag and Drop Models Graph" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
```

Comment:
<img width="1061" alt="Screenshot 2023-06-05 at 9 41 59 PM" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/3248329/20a464db-4947-417e-a7c1-b63d7944b7f4">

There seems to be something off with the rendering here. Wistia logo shows up at the top